vm/src/stdlib_natives.cpp

```diff
@@ -177,7 +187,9 @@ natives_functions_t load_natives(msh::heap &) {
         {"std::set_env", set_env},
         {"std::is_env_def", is_env_def},
         {"std::read_line", read_line},
-        
+
+        {"std::intern::gc", gc},
```

Comment:
```suggestion
        {"std::memory::gc", gc},
```
Intern is for strings.